src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -125,6 +133,7 @@ public async Task RunAsync()
                                               requireExitCodeZero: true,
                                               outputEncoding: outputEncoding,
                                               killProcessOnCancel: false,
+                                              inheritConsoleHandler: inheritConsoleHandler,
```

Comment:
`!ExecutionContext.Variables.Retain_Default_Encoding` like other handler?

Reply:
Whoops, missed this, should be fixed now